internal/cmd/validate.go

```diff
@@ -92,16 +92,16 @@ func validatePreRunE(cmd *cobra.Command, _ []string) error {
 func validateCmdFunc(cmd *cobra.Command, args []string) error {
 	// Initialize variables for multiple files
 	var (
-		onlySchemaFiles            = true
-		validateContentsFiles      = make([][]byte, 0, len(args))
-		parsedFiles                = make([]validationfile.ValidationFile, 0, len(args))
-		totalAssertions            int
-		totalRelationsValidated    int
```

Comment:
All of these are artifacts of the one-file-at-a-time approach that we started with. I've moved them into the loop so that everything is handled on a per-file basis.